ui/src/App.tsx

```diff
@@ -211,6 +217,10 @@ const App = () => {
     const entryId = displayedEventData._id;
     const startTimeUtc = new Date(getDateTimeString(startDate, startTime));
     const endTimeUtc = new Date(getDateTimeString(endDate, endTime));
+    let recurrenceEndUtc;
+    if (recurrenceEnds) {
+      recurrenceEndUtc = new Date(getDateTimeString(recurrenceEnds, startTime));
```

Comment:
Is there a reason that the end date has the startTime prepended rather than the endTime?

https://github.com/jakubroztocil/rrule#usage

Reply:
It seems like `until` doesn't care about the time, so we could remove this for simplicity/clarity of reading. 

https://github.com/jakubroztocil/rrule#usage